extensions/amp-story/1.0/bookend/components/bookend-component-interface.js

```diff
@@ -14,6 +14,30 @@
  * limitations under the License.
  */
 
+/** @enum {string} */
+export const BOOKEND_COMPONENT_CLASSES = {
```

Comment:
Why is this change needed?

Reply:
To fix a circular dependency between `/bookend/bookend-component.js` -> `/bookend/components/{article,portait,...}.js ` -> `/bookend/bookend-component.js`.

I could put them in `amp-story-bookend.js` too if you prefer?

Reply:
Sorry, my question was more around why we need this enum in the first place, over the existing implementation.

Reply:
Doh, sorry. This isn't needed anymore.